src/ui/FormatSize/FormatSize.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '../../testing';
+import { FormatSize } from './FormatSize';
+
+const METRIC_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB'];
+const BINARY_UNITS = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
+
+describe('FormatSize', () => {
+  it('Should format to Bytes', () => {
+    render(<FormatSize prefix="metric">500</FormatSize>);
+    expect(screen.queryByText('500 B')).toBeInTheDocument();
+  });
+
+  it('Should format to KB', () => {
+    render(<FormatSize prefix="metric">1000</FormatSize>);
+    expect(screen.queryByText('1 KB')).toBeInTheDocument();
+  });
+
+  it('Should format to MB', () => {
+    render(<FormatSize prefix="metric">{1000 * 1000}</FormatSize>);
```

Comment:
Вспомнил что можно красиво степени делать:
```suggestion
    render(<FormatSize prefix="metric">{10 ** 6}</FormatSize>);
```